libs/shared/ui/src/lib/components/chart/chart.tsx

```diff
@@ -62,8 +55,14 @@ const ChartTooltipContent = forwardRef<
     title: string
     formatValue?: (value: string, dataKey: string) => string
     formatLabel?: (dataKey: string) => string
+    maxItems?: number
   }
->(({ active, payload, title, formatValue, formatLabel }, ref) => {
+>(function ChartTooltipContent({ active, payload, title, formatValue, formatLabel, maxItems }, ref) {
```

Comment:
I think you can add a value by default 
```suggestion
>(function ChartTooltipContent({ active, payload, title, formatValue, formatLabel, maxItems = 15 }, ref) {
```